templates/R-install-package.j2

```diff
@@ -8,6 +8,7 @@ if (is.null(argv) | length(argv) < 1) {
 
 type = ifelse(is.na(argv[2]), '{{ r_packages_type }}', argv[2]);
 lib = ifelse(is.na(argv[3]), '{{ r_packages_lib }}', argv[3]);
+r_version = '{{ r_version }}';
```

Comment:
No need to define this (it's a `jinja` `if`)

Reply:
Oups, indeed! 
Sorry for that